service/src/main/java/com/commercetools/pspadapter/payone/mapping/PayoneRequestFactory.java

```diff
@@ -37,8 +52,7 @@ public CaptureRequest createCaptureRequest(final PaymentWithCartLike paymentWith
     }
 
     protected void mapFormPaymentWithCartLike(final AuthorizationRequest request,
-                                              final PaymentWithCartLike paymentWithCartLike,
-                                              final Logger logger) {
+                                              final PaymentWithCartLike paymentWithCartLike) {
```

Comment:
Yes, that's what i'm trying to do in the project step by step, but from the beginning it has not have consistent convention, unfortunately, that's why it is still not consistent, but not easy to do this in a single re-factoring step ((